docs/html/topics/configuration.md

```diff
@@ -199,33 +206,38 @@ trusted-host =
 This enables users to add additional values in the order of entry for such
 command line arguments.
 
-## Environment Variables
+(env-variables)=
 
-pip's command line options can be set with environment variables using the
-format `PIP_<UPPER_LONG_NAME>` . Dashes (`-`) have to be replaced with
-underscores (`_`).
+## Environment Variables
 
-- `PIP_TIMEOUT=60` is the same as `--timeout=60`
-- ```
-  PIP_FIND_LINKS="http://mirror1.example.com http://mirror2.example.com"
-  ```
+All pip's command line options have equivalent environment variables,
+they can be specified using:
+```shell
+PIP_<UPPER_LONG_NAME>
+```
 
-  is the same as
+Options with dashes (`-`) have to be replaced with underscores (`_`).
 
-  ```
-  --find-links=http://mirror1.example.com --find-links=http://mirror2.example.com
-  ```
+Examples:
 
-Repeatable options that do not take a value (such as `--verbose`) can be
-specified using the number of repetitions:
+```shell
+PIP_VERBOSE=3 PIP_CACHE_DIR=/home/user/tmp pip ...
+# is equivalent to
+pip ... -vvv --cache-dir=/home/user/tmp
+```
 
-- `PIP_VERBOSE=3` is the same as `pip install -vvv`
+All option values follows the same rules as for the [configuration files](config-file).
 
 ```{note}
 Environment variables set to an empty string (like with `export X=` on Unix) will **not** be treated as false.
 Use `no`, `false` or `0` instead.
+
+Consequently, boolean options prefixed with `--no-*` can be disabled by using
+truthy values, e.g. `PIP_NO_CACHE_DIR=true`.
```

Comment:
I don't know how you want this, I tried to duplicate the general statement, please have a look.

Reply:
Sorry, I meant to include this warning as its own separate thing since "at some point" it'll be fixed, but for now, it's a major footgun that should be called out. I pushed a change to reflect how I wanted it included. Thanks for taking a stab!